financial-templates-lib/test/ExpiringMultiPartyClient.js

```diff
@@ -172,31 +172,6 @@ contract("ExpiringMultiPartyClient.js", function(accounts) {
     ];
     assert.deepStrictEqual(expectedLiquidations.sort(), client.getUndisputedLiquidations().sort());
 
-    // Expire the liquidation by manipulating Date.now to be just after the liquidation's expiry.
-    const expiryTime = (await emp.liquidations(accounts[1], id.toString())).expiry;
```

Comment:
I removed this test because I later test in the "Returns expired liquidations" test that the liquidation is removed from "UndisputedLiquidations" post-liquidation-expiry